pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
Thanks for running the upgrade op (now you know it works) :)

As also mentioned in person is that upgrade operation should be changed as part of this PR because it important to verify that assumptions in upgrade works with this API change (given that this API change is being made for upgrade). 

So while it doesn't break upgrade operation it also doesn't verify that the API change works/is usable/makes sense for upgrade.